frontend/src/scenes/insights/InsightDateFilter/InsightDateFilter.tsx

```diff
@@ -3,25 +3,24 @@ import { useValues, useActions } from 'kea'
 import { insightDateFilterLogic } from './insightDateFilterLogic'
 import { DateFilterProps, DateFilter } from 'lib/components/DateFilter/DateFilter'
 import './index.scss'
+import { insightLogic } from 'scenes/insights/insightLogic'
 
 export function InsightDateFilter(props: DateFilterProps): JSX.Element {
+    const { insightProps } = useValues(insightLogic)
     const {
         dates: { dateFrom, dateTo },
-        highlightDateChange,
-    } = useValues(insightDateFilterLogic)
-    const { setDates } = useActions(insightDateFilterLogic)
+    } = useValues(insightDateFilterLogic(insightProps))
+    const { setDates } = useActions(insightDateFilterLogic(insightProps))
 
     return (
-        <span className={highlightDateChange ? 'insights-date-filter highlighted' : ''}>
```

Comment:
We should also remove this file `frontend/src/scenes/insights/InsightDateFilter/index.scss` as its no longer used

Reply:
thanks, removed!